rust/Cargo.toml

```diff
@@ -2,6 +2,10 @@
 name = "rpmostree-rust"
 version = "0.1.0"
 authors = ["Colin Walters <walters@verbum.org>", "Jonathan Lebon <jonathan@jlebon.com>"]
+# This is kind of a lie, we keep the Rust code MIT / Apache 2.0 but we also
+# directly depend on the systemd crate, and really this is "license of the whole"
+# as opposed "license of the individual code files"
```

Comment:
I think I need some help with this. Why can't we put the MIT/Apache 2 license here? If we theoretically published this crate on crates.io, wouldn't that be the license we'd use? Re. systemd/libdnf, it's fine for MIT apps to link to LGPL libraries, no?

Reply:
So...yeah I kind of tried to do some handwaves :wave: but you caught me :wink: 
Basically I tried setting it to "MIT / Apache 2.0" but cargo-lichking didn't like that and I didn't understand why and didn't want to spend the time on it at the time...

Just marking this one as WIP for now.

Reply:
OK I think actually the checker is right here; we want to make a claim on the *final* license, which is definitely LGPLv2+.